src/grafana_dashboards/prometheus-k8s_rev1.json.tmpl

```diff
@@ -0,0 +1,2972 @@
+{
```

Comment:
Should the file be named `self-monitoring_rev1`?

Reply:
Seems redundant, since it's Prometheus providing its own dashboards.... anyway I have no strong preferences... we should follow the same pattern in all charms.
What do you think @simskij ?